testsuite/features/support/env.rb

```diff
@@ -85,7 +85,7 @@ def enable_assertions
 # embed a screenshot after each failed scenario
 After do |scenario|
   current_epoch = Time.new.to_i
-  STDOUT.puts "This scenario took: #{current_epoch - @scenario_start_time} seconds"
+  log "This scenario took: #{current_epoch - @scenario_start_time} seconds"
```

Comment:
So we can use `log` here because we are in a Cucumber scope, correct?

Reply:
Right

Reply:
I was not aware of it, until I tested...